<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carbonwickler_main</class>
 <widget class="QMainWindow" name="carbonwickler_main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>carbonwickler_main</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ButtonOpenFile">
    <property name="geometry">
     <rect>
      <x>53</x>
      <y>440</y>
      <width>131</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Datei auswählen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonGenCode">
    <property name="geometry">
     <rect>
      <x>233</x>
      <y>440</y>
      <width>121</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>g-code erstellen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonQuit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>440</y>
      <width>131</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Pr0gramm beenden</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="d_end">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="l_gesamt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>114</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Gesamtlänge (mm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>114</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Durchmesser (mm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>123</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Endstücklänge (mm)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>93</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Wickelwinkel (°)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>87</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Lagen (max. 5)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="lagen">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSlider" name="winkel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
    <property name="singleStep">
     <number>5</number>
    </property>
    <property name="pageStep">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePath">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>361</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="l_end">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPr0gramm">
    <property name="title">
     <string>&amp;Pr0gramm</string>
    </property>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuPr0gramm"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBeenden">
   <property name="text">
    <string>&amp;Beenden</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
